plugin/start-console.sh

```diff
@@ -2,7 +2,8 @@
 
 set -euo pipefail
 
-CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
+OPENSHIFT_VERSON=$(oc version | grep "Server Version: " | awk '{print $3}' | cut -d. -f-2)
```

Comment:
just FYI: this assumes (and will only work) if you are already logged into an OpenShift cluster. Otherwise, "oc version" won't tell you about a "Server Version" :
```
$ oc version
Client Version: 4.13.0-202305312300.p0.g05d83ef.assembly.stream-05d83ef
Kustomize Version: v4.5.7
```

and the rest of this script will fail somewhere along the way.

I would suggest doing something like this at the top on line 4: https://github.com/kiali/kiali/blob/master/Makefile#L228